src/FsAutoComplete.Core/CommandResponse.fs

```diff
@@ -248,6 +264,11 @@ module CommandResponse =
 
   let info (serialize : Serializer) (s: string) = serialize { Kind = "info"; Data = s }
   let error (serialize : Serializer) (s: string) = serialize { Kind = "error"; Data = s }
```

Comment:
@Krzysztof-Cieslak and @rneatherway , same issue: generic error

> Right, I see. What code do you get for a generic error that we don't have more information for?

I dont want to change the old contract for error, or replace it completly, is too much a breaking change.

For example for `project` with generic error, will continue to return usual `{ Kind: "error", Data: "error message" }`, just for known error use the detailed `{ Kind: "error", Data: { Code: 101: Message: "dadsa", AdditionalData: { ... } }`.

this is ok ihmo, so current consumer code (your awesome exnntesion) continue to work the same.

pratically error is always

```
{ Kind: "error", Data: <error_data>  }
```

and `<error_data>` is or `string` or `ResponseError<'T>`. if was a du:

```fsharp
type ErrorData =
   | Generic of string
   | Detailed of ResponseError<'T>
```


@rneatherway @Krzysztof-Cieslak do you think we need anyway a generic error code? np for me, just want to explain why was like that